chaosgcp/dns/actions.py

```diff
@@ -0,0 +1,67 @@
+"""DNS A record update.
+
+This file has the function to update the DNS A record
+
+Typical usage example in an experiment json file:
+
+  "module": "chaosgcp.dns.actions",
+  "func": "update_service",
+"""
+
+# Copyright 2023 Google LLC.
+# SPDX-License-Identifier: Apache-2.0
+# -*- coding: utf-8 -*-
+from typing import Any, Dict
+from chaosgcp import client
+from chaoslib.types import Secrets
+
+__all__ = ["update_dns_a_record"]
+
+
+def update_dns_a_record(
```

Comment:
considering you take a `record_type` as a parameter, this function should only be named `update_dns_record`. Or, remove the parameter since it cannot be anything but a `A` record anyway.